wp-content/themes/twentyseventeen-wp20/page-whats-on.php

```diff
@@ -0,0 +1,25 @@
+<?php get_header(); ?>
+
+<div class="wrap">
+	<div id="primary" class="content-area">
+		<main id="main" class="site-main" role="main">
+			<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+
+				<header class="entry-header">
+					<h1 class="entry-title">
+						<?php esc_html_e( 'People all over the world are celebrating the WordPress 20th Anniversary on May 27, 2022. Join us!', 'wp20' ); ?>
```

Comment:
Hi! I think this should be May 27, 2023 instead of 2022? @adamwoodnz

Reply:
Hey 👋🏻 You're right, thanks for pointing that out. @timse201 fixed that in #25